### DISCUSSION TITLE
How to Remove Previous Years' Contributions from Your GitHub Profile?

### DISCUSSION THREAD

**Discussion by alidehkhodaei** [community/community#55153] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,

I recently opened an old project that had commits from 2016 and pushed it to my GitHub repository. However, when the project was pushed, my GitHub profile displayed that I had contributions in 2016, 2017, 2018, and 2019, even though I only joined GitHub in 2020. Can you please guide me on how to remove these four years from my profile? I have deleted the project, but these years are still showing up.

Thank you.

**Comment by minhajul-im** [comment]
exactly it was my question. unfortunately, i can't find that answer.

**Comment by hunhee98** [comment]
How did you solve it?

**Comment by alidehkhodaei** [comment]
@minhajul-im I'm sorry, I don't know why I'm seeing your message now.

**Reply by minhajul-im** [reply]
okay, no problem. thanks

**Comment by alidehkhodaei** [comment, marked as answer]
I opened a ticket with GitHub support, explained my issue, and it got resolved.

**Reply by hunhee98** [reply]
Thanks, I'll try same way

**Reply by CryptoNinja0331** [reply]
> I opened a ticket with GitHub support, explained my issue, and it got resolved.

thanks. I met that same issue.

**Reply by CryptoNinja0331** [reply]
I will try

**Reply by riley-dickerson** [reply]
Oh, great.
I also have the same issue from my funny mistake, fake years are not displayed in my profile.
But others can see it, and I do not want it.
If I delete the repository, it's fake year will be removed automatically after 90 days?
Without helping from the github support team.
How do you think?

**Reply by alidehkhodaei** [reply]
> If I delete the repository, it's fake year will be removed automatically after 90 days? Without helping from the github support team. How do you think?

Hello, I haven't tested this item.

**Reply by junglerot** [reply]
What ticket did you open?

**Reply by alidehkhodaei** [reply]
Explain the problem to them, and they will fix it.
